server/belga/io/feed_parsers/base_belga_newsml_1_2.py

```diff
@@ -227,7 +251,7 @@ def parser_identification(self, item, indent_el):
         if newsident_el is not None:
             element = newsident_el.find('ProviderId')
             if element is not None:
-                item['provider_id'] = element.text
+                item['original_source'] = element.text
```

Comment:
Because, its values like describing a original_source (www.itar-tass.com, texto.efeservicios.com). and I don't using provider_id, because I don't want to misunderstand with provider_id of ingest.
And should we consider it as CV?

Reply:
IMO no need in CV for it. Let's use `provider_id` key for it.